<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindowClass</class>
 <widget class="QMainWindow" name="mainwindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainwindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>941</width>
      <height>461</height>
     </rect>
    </property>
    <property name="title">
     <string>文件列表</string>
    </property>
    <widget class="QTableWidget" name="list_file">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>921</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>967</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_connect">
    <property name="title">
     <string>连接</string>
    </property>
    <addaction name="action_connect"/>
   </widget>
   <widget class="QMenu" name="menu_quit">
    <property name="title">
     <string>退出</string>
    </property>
    <addaction name="action_quit"/>
   </widget>
   <addaction name="menu_connect"/>
   <addaction name="menu_quit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_connect">
   <property name="text">
    <string>连接</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
